src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -546,6 +554,11 @@ package com.videojs.providers{
                     break;
 
                 case "NetStream.Buffer.Full":
+                    if (_isSeeking) {
```

Comment:
Very nice. It's probably worth a code comment here so we don't have to dig up the issue later to know why this is here instead of Seek.Notify.